### ISSUE TITLE
Broward County FL us/fl/broward

### ISSUE THREAD

**Issue by ssilverm** [openaddresses/openaddresses#4163]
Hey All,

Found an issue with the Broward County FL sources.

http://results.openaddresses.io/sources/us/fl/broward

On 2017-09-04 **759,532** addresses were available (including my grandparents).  After that date, the available addresses dropped to **480,700**.

Testing with my grandparent's address, it no longer appears in the source csv correctly.

Not sure how to fix it, but I'm personally reverting to using the old broward.csv for now.

Thank you for all that you do!

**Comment by justinelliotmeyers** [comment]
we went from their rest service to a direct download. I'll reach out and see what is going on with the data. Thanks for pointing out.

**Comment by ssilverm** [comment]
Thank you!  Interested to hear what you find.

**Comment by justinelliotmeyers** [comment]
477,759 records in raw data here: https://bcgis.broward.org/GISData/Zipfiles/GISaddresspoints.zip

I'll call them and find out or check the previous source to see what happened, or scope of extent of data. What is the dl of the source of the old data so I can plot and compare.  Thanks

**Comment by ssilverm** [comment]
✔ | 759,532 | 2017-09-04 | 80c7db1 (0:05:35) | 438d346 (0:11:08) | 904db83 | 6.4.1
-- | -- | -- | -- | -- | -- | --

Data: https://s3.amazonaws.com/data.openaddresses.io/runs/251954/us/fl/broward.zip


https://s3.amazonaws.com/data.openaddresses.io/runs/251954/output.txt

https://github.com/openaddresses/openaddresses/blob/1e5b4b1af119da3209c15d6665cca42ad21f4cf7/sources/us/fl/broward.json

https://github.com/openaddresses/machine/releases/tag/6.4.1

**Comment by justinelliotmeyers** [comment]
Thanks!!

**Comment by justinelliotmeyers** [comment]
called, left a voicemail:

Planning and Development Management Division


Address: 1 N. University Dr. - Mailbox 102A - Plantation, FL 33324

Email: PDMDinfo@broward.org​

Tel: 954-357-6634

**Comment by justinelliotmeyers** [comment]
yeah, the data must have an issue; they are missing tons of locations. Downloading the raw parcel data from here: ftp://sdrftp03.dor.state.fl.us/Map%20Data/00_2018/

To see if this is a better secondary source

**Comment by justinelliotmeyers** [comment]
missing multifamily, condos, and appartments. She is in the process of creating. She is retiring in 2-3 years - only person working on it. She doesn't know about the rest services.

She is uploading the new/ full dataset to a FTP today.  I'll keep you posted and hopefully we can regain access to the rest services.

Stay Tuned!

**Comment by justinelliotmeyers** [comment]
https://services.arcgis.com/JMAJrTsHNLrSsWf5/arcgis/rest/services/OpenDataAddressPoints/FeatureServer

https://services.arcgis.com/JMAJrTsHNLrSsWf5/arcgis/rest/services/PointAddressesCityApp/FeatureServer

 

 

 

Hi Justin, as we discussed here are the urls for the two address feature services

The first is mostly residential single family. These points are placed on the actual location of where the address is.

The second is based on the property appraiser situs address and it is parcel poly centroids ; There is a point for every parcels but for condominiums they draw on top of each other. If you look at the circled points they look like 3 but the selection shows 248 points

There are also points that don’t have addresses.

I hope this helps
![image003](https://user-images.githubusercontent.com/5324316/44928141-e65bbd80-ad24-11e8-8dd7-6d5f02f9b235.jpg)

 



 

 

https://services.arcgis.com/JMAJrTsHNLrSsWf5/arcgis/rest/services/OpenDataAddressPoints/FeatureServer

https://services.arcgis.com/JMAJrTsHNLrSsWf5/arcgis/rest/services/PointAddressesCityApp/FeatureServer

**Comment by justinelliotmeyers** [comment]
@ssilverm 
this has 808k
https://services.arcgis.com/JMAJrTsHNLrSsWf5/ArcGIS/rest/services/PointAddressesCityApp/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=&returnGeometry=true&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=true&returnExtentOnly=false&returnDistinctValues=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=html&token=
